openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphStorageFactory.java

```diff
@@ -28,7 +28,7 @@
 
 import com.graphhopper.routing.weighting.Weighting;
 
-public class ORSGraphStorageFactory implements GraphStorageFactory {
+public class 		ORSGraphStorageFactory implements GraphStorageFactory {
```

Comment:
Done

Reply:
Thanks for your attempt. Unfortunately, now the whole file's content shows up as modiefied. I believe there should be no changes to this file at all. In general, unless intended let's try to avoid such spurious modifications obscuring the actual content of PRs. Thanks!

Reply:
I used the auto-formatter to reformat the file. This formatted a lot of lines. I guess there is no consistency in general on how our code is formatted. I can rever this part and remove the wrong indentation that you mentioned in the first place.

Reply:
Redone to just remove the single wrong indentation